### DISCUSSION TITLE
How to change the style/class of the selectcolumn ?

### DISCUSSION THREAD

**Discussion by marcelluswalace** [Comcast/react-data-grid#2780] [General]
Good morning,
i've been struggling to find how to change the look and feel of the checkbox of the column SelectColumn which add the ability to select a row. Can someone give me some indications?

const columns: Col[] = [
  SelectColumn,    // <- how to style this? is it even possible?
  { key: 'status', name: t('patate'), width: 70 },
]; 

Regards

**Comment by NickyAlfEdge** [comment]
I understand that this is old but for people who are curious, use the spread operator:

```javascript
{
  ...SelectColumn,
  headerCellClass: 'header-cell-class',
  cellClass: 'cell-class',
},
```
